src/libstore-tests/package.nix

```diff
@@ -79,7 +80,7 @@ mkMesonExecutable (finalAttrs: {
               mkdir -p "$HOME"
             ''
             + ''
-              export _NIX_TEST_UNIT_DATA=${data + "/src/libstore-tests/data"}
+              export _NIX_TEST_UNIT_DATA=${resolvePath ./data/src/libstore-tests/data}
```

Comment:
This worked in Nixpkgs, but not here. Might need a more explicit conditional, since we don't have the "source layer" to do the work for us in this case.

Reply:
Or expose the "source layer" logic (or similar) as an independent function.